.github/workflows/dev-release.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
   dev-build-deb:
     name: "Development Release Deb"
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
```

Comment:
does the deb package build on ubuntu-22.04 compatible on lower version ubuntu? Many users are using ubuntu 18.04 for now

Reply:
+1, we need the binary to run on 18.04 :)

Reply:
Now we use Ubuntu-18.04 container run on 22.04 to build deb package.